data_structures/delete_node_without_head.cpp

```diff
@@ -0,0 +1,67 @@
+#include <bits/stdc++.h>

```

Comment:
https://www.geeksforgeeks.org/delete-a-node-from-linked-list-without-head-pointer/
I really feel it copied from here. 
And such a type of code is not acceptable. Please read contributing guidelines.

Reply:
Can you please give an overview of what type of changes you want me to be made in this code?

Reply:
https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md

Reply:
Kindly review again, I comitted some necessary changes.